site/_posts/2020-07-20-breached-password-detection.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog-post
+title: How to prevent your users from using breached passwords
+description: In this tutorial, we'll build a basic Django web application using FusionAuth for an easier and safer way of handling user registration and authentication.
+author: Dan Moore
+image: blogs/social-sign-in-django/headerimage.png
+category: blog
+tags: client-python
+excerpt_separator: "<!--more-->"
+---
+
+While there are many ways to authenticate against online systems, usernames such as email addresses and passwords are still commonly used credentials. Unfortunately, many passwords have been compromised and made available on the Internet. When combined with the fact that users often reuse passwords across different systems, this means that your application or site may be at risk through no fault of your own.
+
+<!--more-->
+
+## The danger of compromised passwords
+
+Compromised passwords allow unauthorized access to your systems. If a user's credentials are known to another party, that party can access your systems acting as that user. Depending on data and functionality available in your systems, the results of unauthorized access could range from worrying to disastrous.
+
+## How to avoid unauthorized access
```

Comment:
'How to protect against breached passwords"

Reply:
Hmmm. I cover more than that in this section, including things like MFA. So I'm not sure this is appropriate. But I can add another section header. I'll do that.